i18n/en.json

```diff
@@ -296,6 +296,22 @@
   "admin.elasticsearchStatus.statusRequestCancel": "Canceling Job...",
   "admin.elasticsearchStatus.statusSuccess": "Indexing complete.",
   "admin.elasticsearchStatus.statusSuccess.help": "Indexing is complete and new posts are being automatically indexed.",
+  "admin.jobTable.jobId": "Job ID: ",
+  "admin.jobTable.statusPending": "Pending",
+  "admin.jobTable.statusInProgress": "In Progress",
+  "admin.jobTable.statusSuccess": "Success",
+  "admin.jobTable.statusError": "Error",
+  "admin.jobTable.statusCanceling": "Canceling...",
+  "admin.jobTable.statusCanceled": "Canceled",
+  "admin.jobTable.lastActivityAt": "Last Activity: ",
+  "admin.jobTable.runLengthSeconds": " seconds",
+  "admin.jobTable.runLengthMinutes": " minutes",
+  "admin.jobTable.cancelButton": "Cancel",
+  "admin.jobTable.headerStatus": "Status",
+  "admin.jobTable.headerFinishAt": "Finish At",
```

Comment:
0/5, would we wanna call this `Sync Finish Time` to clarify it's when the AD/LDAP sync was finished?

(I was also considering `Sync Completion Time` but sometimes that's inaccurate since the sync may not have completed)

Reply:
The table is generic, for all job types, so it either needs customisable column headings or it the headings need to stay generic as they are now.

Reply:
I see, got it. Maybe `Finish Time` then? I think it might be easier to understand for non-English people than `Finish At`.

Reply:
0/5 I'll switch to finish time.